<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addBidderWindow</class>
 <widget class="QMainWindow" name="addBidderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Bidder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="firstNameLabel">
      <property name="text">
       <string>First name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="firstNameInput"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="surnameLabel">
      <property name="text">
       <string>Surname</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="surnameInput"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="applyButton">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="response">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
